charts/kube-prometheus-stack/values.yaml

```diff
@@ -2377,8 +2377,17 @@ prometheus-node-exporter:
     labels:
       jobLabel: node-exporter
 
+  ## Enable scraping via kubernetes-service-endpoints
+  ## Disabled by default as we service monitor is enabled below
+  ##
+  prometheusScrape: false
```

Comment:
Please double check this.

The values are below `prometheus-node-exporter` and not related to kube-state-metrics

Reply:
Good catch, I tried doing this via the gh website so I didn't have to clone locally but ran into weird permission issues so was about the third time I'd updated and obviously didn't check it properly.

Updated in correct place now.